libs/execution_context/ebpf_program.c

```diff
@@ -11,7 +11,7 @@
 typedef struct _FILE FILE;
 #include "ubpf.h"
 
-typedef struct _ebpf_program
+typedef struct bpf_program
```

Comment:
because this file includes ebpf_link.h which has
```
    typedef struct bpf_program ebpf_program_t;
```
So if it's not renamed, the types conflict and this results in a compiler error.

Reply:
I changed this and ebpf_link.h back, they just have to match each other and don't need to use a libbpf type since this is internal.